src/packagers/SourceMapPackager.js

```diff
@@ -16,7 +16,7 @@ class SourceMapPackager extends Packager {
 
   async end() {
     let file = path.basename(this.bundle.name);
-    await this.write(this.sourceMap.stringify(file));
+    await this.write(this.sourceMap.stringify(file, this.options.rootDir));
```

Comment:
Should the sourceRoot be a relative path rather than absolute? seems like maybe `path.relative(options.outDir, options.rootDir)`?

Reply:
Updated to relative from outDir path